docs/serverless-demo.md

```diff
@@ -42,7 +40,7 @@ query RootWebsiteQuery($offset: Int, $limit: Int) {
 }
 ```
 
-The Apollo Router also supports [GraphQL subscriptions](https://www.apollographql.com/docs/graphos/operations/subscriptions/)!
+The [Apollo Router also supports GraphQL subscriptions](https://www.apollographql.com/docs/graphos/operations/subscriptions/)!
```

Comment:
Possibly add something about what this operation is doing? I'm actually not sure what this was doing / what it's listening for.